tools/ports/__init__.py

```diff
@@ -393,10 +396,24 @@ def add_deps(node):
     add_deps(port)
 
 
-def handle_use_port_arg(settings, name):
+def handle_use_port_arg(settings, arg):
+  args = arg.split('?', 1)
+  name, options = args[0], args[1] if len(args) == 2 else None
```

Comment:
Doesn't this have to be (None, None) on the right?

Reply:
No => this reads:

`name` gets assigned to `args[0]`
`options` gets assigned to `args[1] if len(args) == 2 else None`

Reply:
I can split in 2 lines if you think that is clearer

Reply:
Great thanks.  I've never like that one line `if/`else` syntax.